Bottom: 390a8bebb797b1be990bab9696416c1107670df4
Top:    3e3fc750b0aebaa1a1c56d0c8cc8ea31b4e8714e
Author: scott snyder <sss@karma>
Date:   2020-02-12 03:32:12 -0500

Fix compilation warnings.


---

diff --git a/src/DebugDigiProcessor.cc b/src/DebugDigiProcessor.cc
index 62f9c8f..312966e 100644
--- a/src/DebugDigiProcessor.cc
+++ b/src/DebugDigiProcessor.cc
@@ -112,7 +112,7 @@ void DebugDigiProcessor::modifyEvent( LCEvent * evt ) {
       
       col = evt->getCollection( _hitColNames[ i ]  ) ;
     }
-    catch( DataNotAvailableException ) {}
+    catch( const DataNotAvailableException& ) {}
 
     if( col ) {
 
@@ -133,7 +133,7 @@ void DebugDigiProcessor::modifyEvent( LCEvent * evt ) {
       
     col = evt->getCollection( _trkColName ) ;
   }
-  catch( DataNotAvailableException ) {}
+  catch( const DataNotAvailableException& ) {}
 
   if( col ) {
